packages/manager/modules/pci/src/projects/project/ai-dashboard/ai-dashboard.routing.js

```diff
@@ -5,21 +5,23 @@ export default /* @ngInject */ ($stateProvider) => {
     url: '/ai-dashboard',
     component: 'ovhManagerPciProjectAiDashboard',
     redirectTo: (transition) =>
-      transition
-        .injector()
-        .getAsync('aiItems')
-        .then((items) =>
-          countAiItems(items) === 0
-            ? { state: 'pci.projects.project.ai-dashboard.onboarding' }
-            : { state: 'pci.projects.project.ai-dashboard.home' },
-        ),
+      Promise.all([
```

Comment:
**nitpick:** Since we are still using angularjs, it is advisable to use `$q` instead of native `Promise` as there is a chance that angularjs may not be notified when the promises are resolved.

Reply:
Changed here: https://github.com/ovh/manager/pull/8818/commits/2eb0297f01171932eafd67126a9432eda8cc4072